charts/incubator/splunk/SCALE/questions.yaml

```diff
@@ -0,0 +1,667 @@
+groups:
```

Comment:
This file needs to be completely redone, we completely overhauled these files today with more features.